tests/test_tax.py

```diff
@@ -5936,10 +6045,6 @@ def test_metagenome_LIN_lingroups(runtmp):
     print(c.last_result.err)
 
     assert c.last_result.status == 0
-    assert (
```

Comment:
I don't quite understand why this is removed as output - I think it's ok, just curious about the behavioral change.

Reply:
This originally printed once for every query that is being summarized .. which means the logging is ~10k lines of this. It made it very difficult to see the actual errors.

It probably would be helpful to print it just once, but then I'd need to keep track of whether or not it had been printed. Will think on it.

Reply:
no need, must have just missed where it was edited out in my quick scan of the code. all good to me!

Reply:
got it. I just updated the description to ...be more descriptive about this :)

Reply:
(please merge ;)